packages/@n8n/api-types/src/dto/auth/login-request.dto.ts

```diff
@@ -2,7 +2,12 @@ import { z } from 'zod';
 import { Z } from 'zod-class';
 
 export class LoginRequestDto extends Z.class({
-	email: z.string().email(),
+	/*
+	 * The LDAP username does not need to be an email, so email validation
+	 * is not enforced here. The controller determines whether this is an
+	 * email and validates when LDAP is disabled
+	 */
+	emailOrLdapUsername: z.string().trim(),
```

Comment:
Is it worth calling this `ldapUid` to match the LDAP configuration. This maps to the `user identified` / `uid` right?